app/client/ui/WelcomeQuestions.ts

```diff
@@ -54,21 +57,21 @@ export function showWelcomeQuestions(userPrefsObs: Observable<UserPrefs>) {
 }
 
 const choices: Array<{icon: IconName, color: string, text: string}> = [
-  {icon: 'UseProduct', color: `${colors.lightGreen}`, text: 'Product Development' },
-  {icon: 'UseFinance', color: '#0075A2',       text: 'Finance & Accounting'},
-  {icon: 'UseMedia',   color: '#F7B32B',       text: 'Media Production'    },
-  {icon: 'UseMonitor', color: '#F2545B',       text: 'IT & Technology'     },
-  {icon: 'UseChart',   color: '#7141F9',       text: 'Marketing'           },
-  {icon: 'UseScience', color: '#231942',       text: 'Research'            },
-  {icon: 'UseSales',   color: '#885A5A',       text: 'Sales'               },
-  {icon: 'UseEducate', color: '#4A5899',       text: 'Education'           },
-  {icon: 'UseHr',      color: '#688047',       text: 'HR & Management'     },
-  {icon: 'UseOther',   color: '#929299',       text: 'Other'               },
+  {icon: 'UseProduct', color: `${colors.lightGreen}`, text: t('ProductDevelopment') },
+  {icon: 'UseFinance', color: '#0075A2',       text: t('FinanceAccounting')},
+  {icon: 'UseMedia',   color: '#F7B32B',       text: t('MediaProduction')    },
+  {icon: 'UseMonitor', color: '#F2545B',       text: t('ITTechnology')     },
+  {icon: 'UseChart',   color: '#7141F9',       text: t('Marketing')           },
+  {icon: 'UseScience', color: '#231942',       text: t('Research')            },
+  {icon: 'UseSales',   color: '#885A5A',       text: t('Sales')               },
+  {icon: 'UseEducate', color: '#4A5899',       text: t('Education')           },
+  {icon: 'UseHr',      color: '#688047',       text: t('HRManagement')     },
+  {icon: 'UseOther',   color: '#929299',       text: t('Other')               },
```

Comment:
```suggestion
  {icon: 'UseProduct', color: `${colors.lightGreen}`, textKey: 'ProductDevelopment'  },
  {icon: 'UseFinance', color: '#0075A2',              textKey: 'FinanceAccounting'   },
  {icon: 'UseMedia',   color: '#F7B32B',              textKey: 'MediaProduction'     },
  {icon: 'UseMonitor', color: '#F2545B',              textKey: 'ITTechnology'        },
  {icon: 'UseChart',   color: '#7141F9',              textKey: 'Marketing'           },
  {icon: 'UseScience', color: '#231942',              textKey: 'Research'            },
  {icon: 'UseSales',   color: '#885A5A',              textKey: 'Sales'               },
  {icon: 'UseEducate', color: '#4A5899',              textKey: 'Education'           },
  {icon: 'UseHr',      color: '#688047',              textKey: 'HRManagement'        },
  {icon: 'UseOther',   color: '#929299',              textKey: 'Other'               },
```

We need to defer translation here also. Storing just the translation keys and calling `t()` where needed is one approach that works.